amd64.env

```diff
@@ -1,7 +1,7 @@
-rosdistro=galactic
+rosdistro=humble
 rmw_implementation=rmw_cyclonedds_cpp
-base_image=ubuntu:20.04
-cuda_base_image=nvidia/cuda:11.6.2-devel-ubuntu20.04
+base_image=ubuntu:22.04
+cuda_base_image=ubuntu:22.04
```

Comment:
@kenji-miyake 
Currently following have only the AMD64 for now:
- https://gitlab.com/nvidia/container-images/cuda/blob/master/dist/11.8.0/ubuntu2204/devel/cudnn8/Dockerfile

Once they also have both ARM64 and AMD64 like it was in the old container, we can switch right?
- https://gitlab.com/nvidia/container-images/cuda/blob/master/dist/11.8.0/ubuntu2004/devel/cudnn8/Dockerfile

Reply:
Yes, I believe so. We can update here in https://github.com/autowarefoundation/autoware/pull/3084.

Reply:
FYI: https://gitlab.com/nvidia/container-images/cuda/-/issues/180#note_1191984633